library/src/utils/text.ts

```diff
@@ -4,19 +4,20 @@ export const isBoolString = (str: string) => str.trim() === 'true'
 
 export const kebab = (str: string) =>
   str
-    .replace(/([a-z0-9])([A-Z])/g, '$1-$2')
-    .replace(/([a-z])([0-9]+)/gi, '$1-$2')
-    .replace(/([0-9]+)([a-z])/gi, '$1-$2')
+    .replace(/([A-Z])/g, '-$1')
```

Comment:
I'm not quite sure what all of this was for. This is where I potentially broke something, but it seemed fine in my tests. I just made this the inverse of the camel case function so things round trip.